### ISSUE TITLE
文献带读 | STAligner

### ISSUE THREAD

**Issue by ixxmu** [ixxmu/mp_duty#4986]
https://mp.weixin.qq.com/s/pVYLIkqI48DpWFJSEekoxA

**Comment by ixxmu** [comment]
文献带读 | STAligner by 单细胞训练营
------
<div><section data-tool="mdnice编辑器" data-website="https://www.mdnice.com" data-mpa-powered-by="yiban.io"><p data-tool="mdnice编辑器">[[----------]]</p><h1 data-tool="mdnice编辑器"><span>Integrating spatial transcriptomics data across different conditions, technologies and developmental stages<span></span></span></h1><h2 data-tool="mdnice编辑器"><span></span><span>[[Xiang Zhou]]; [[Kangning Dong]]; [[Shihua Zhang]]</span><span></span></h2><h2 data-tool="mdnice编辑器"><span></span><span>Abstract</span><span></span></h2><p data-tool="mdnice编辑器">With the rapid generation of spatial transcriptomics (ST) data, integrative analysis of multiple ST datasets from different conditions, technologies and developmental stages is becoming increasingly important. Here we present a graph attention neural network called STAligner for integrating and aligning ST datasets, enabling spatially aware data integration, simultaneous spatial domain identification and downstream comparative analysis. We apply STAligner to ST datasets of the human cortex slices from different samples, ==the mouse olfactory bulb slices generated by two profiling technologies==, the mouse hippocampus tissue slices under normal and Alzheimer’s disease conditions, and the spatiotemporal atlases of mouse organogenesis. STAligner efficiently captures the shared tissue structures across different slices, the disease-related substructures and the dynamical changes during mouse embryonic development. In addition, the shared spatial domain and ==nearest-neighbor pairs identified by STAligner can be further considered as corresponding pairs to guide the threedimensional reconstruction of consecutive slices==, achieving more accurate local structure-guided registration than the existing method.</p><p><img data-galleryid="" data-imgfileid="100000951" data-ratio="0.5583333333333333" data-s="300,640" data-src="https://mmbiz.qpic.cn/mmbiz_png/LjkGSkc5VwIgicG5W2sEU9UuWzL9U0x2JepMC9ZWdGtibRkJVd5Cncj3d96ic9g4Aw1N2yKGCOK8oP00bicoy8uyfw/640?wx_fmt=png&amp;from=appmsg" data-type="png" data-w="1080" src="https://mmbiz.qpic.cn/mmbiz_png/LjkGSkc5VwIgicG5W2sEU9UuWzL9U0x2JepMC9ZWdGtibRkJVd5Cncj3d96ic9g4Aw1N2yKGCOK8oP00bicoy8uyfw/640?wx_fmt=png&amp;from=appmsg"></p><br><h2 data-tool="mdnice编辑器"><span></span><span>Quote</span><span></span></h2><blockquote data-tool="mdnice编辑器"><span></span><p>STAligner aims to identify consensus spatial domains across slices, which is different from the goal of the non-integration methods BayesSpace, SpaGCN10 and STAGATE12</p></blockquote><h2 data-tool="mdnice编辑器"><span></span><span>Key points</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>In multicellular organisms, cells are deeply influenced by their tissue environment</section></li><li><section>STAligner defines the triplet that contains anchor– positive and anchor–negative spot pairs. The former is defined as the mutual nearest neighbors (MNNs) with similar gene expressions but belong to two different slices, and the latter belongs to the same slice with different spatial positions and dissimilar expressions</section></li><li><section>By taking the manual annotations as ground truth, we evaluated the performance of STAligner on simultaneous identification of spatial domains and compared it with that of the non-spatial integration method Harmony and two recently developed spatial integration approaches SEDR and PASTE in terms of the adjusted Rand index (ARI) (Methods)</section></li><li><section>The six shared spatial domains by the two slices were spatially ordered from the inner layers, middle layers to outer layers of the mouse olfactory bulb, including the rostral migratory stream (RMS), granule cell layer (GCL), mitral cell layer (MCL), external plexiform layer (EPL), glomerular layer (GL) and olfactory nerve layer (ONL) (Fig. 3c–e)</section></li><li><section>STAligner aims to identify consensus spatial domains across slices, which is different from the goal of the non-integration methods BayesSpace[^11], SpaGCN10 and STAGATE12</section></li><li><section>We demonstrate improved performance of STAligner on diverse datasets from different individuals, technologies, disease conditions and developmental stages</section></li><li><section>Our 3D reconstruction strategy based on the Iterative Closest Point (ICP) algorithm is limited to rigid transformation and cannot account for local flexible distortions</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Summary</span><span></span></h2><h3 data-tool="mdnice编辑器"><span></span><span>Introduction</span><span></span></h3><p data-tool="mdnice编辑器">Cells are deeply influenced by their tissue environment. Knowledge of the spatial position of cells has a critical role in understanding why cells from different locations have distinctive structures and functions[^1].Slide-seq improves spatial resolution to near-cellular level (10 μm)[^4],[^5] and Stereo-seq further reaches subcellular resolution (0.22 μm)[^6]These technologies have been widely used to explore the biological architecture of tissues[^4],[^5],[^7], the biomarkers of complex disease[^8] and the development process of organogenesis[^6].Computational methods have been successfully applied to ST data of a single condition, technology or developmental stage to characterize the tissue structure[^9],[^10],[^11],[^12], identify spatially variable genes[^13],[^14] and decipher spatial cell–cell communication patterns[^15].Integrated analysis of ST datasets produced by different conditions, technologies or developmental stages remains challenging.Forebrain Hindbrain Hypothalamus Olfactory bulb Meninges Branchial arch Jaw and tooth Surface ectoderm Dorsal root ganglion</p><h3 data-tool="mdnice编辑器"><span></span><span>Objectives</span><span></span></h3><p data-tool="mdnice编辑器">The authors aim to enforce the encoder to learn a batchcorrected embedding for multiple ST data via the triplet learning, which has shown superior performance in diverse fields including scRNA-seq dataset integration</p><h3 data-tool="mdnice编辑器"><span></span><span>Methods</span><span></span></h3><p data-tool="mdnice编辑器">Data preprocessing STAligner takes gene expression and spatial coordinate information of multiple ST slices as input.The authors select the top-5,000 highly variable genes.The authors take the intersection of the highly variable genes of all the input slices to ensure the spot features are shared across slices.Construction of the spatial neighbor graph On the basis of the spatial coordinate information of all spots, the authors measure the similarities between spots by the Euclidean distance.Two spots are considered as neighbor if the distance between them is shorter than a pre-defined threshold r.Using the spatial neighbor relationship, the authors construct an undirected neighbor graph denoted as the adjacency matrix A, where Aij = 1 if and only if there is an edge between spot i and spot j.The authors can adjust r so that each spot has 4–15 nearest neighbors on average for diverse ST scenarios and platforms</p><h3 data-tool="mdnice编辑器"><span></span><span>Results</span><span></span></h3><p data-tool="mdnice编辑器">Overview of STAligner STAligner takes the normalized gene expression and spatial coordinates of ST data from multiple tissue slices as input (Fig. 1).For each ST slice, STAligner constructs a spatial neighbor graph between spots using the spatial coordinates.STAligner employs a graph attention autoencoder neural network to learn spatially aware embeddings based on the spatial neighbor graph, and constructs the spot triplets based on current embeddings to guide the integration and alignment process of different slices by attracting similar spots and discriminating dissimilar spots across slices.STAligner defines the triplet that contains anchor– positive and anchor–negative spot pairs.The former is defined as the mutual nearest neighbors (MNNs) with similar gene expressions but belong to two different slices, and the latter belongs to the same slice with different spatial positions and dissimilar expressions.The graph attention autoencoder training and triplet construction are optimized iteratively until batch-corrected embeddings are generated</p><p><img data-galleryid="" data-imgfileid="100000952" data-ratio="1.0074074074074073" data-s="300,640" data-src="https://mmbiz.qpic.cn/mmbiz_png/LjkGSkc5VwIgicG5W2sEU9UuWzL9U0x2Jt8Rsu3NTib2l0arlJQZ1ib7Ed1ag7czYnNChxPFtZOictFhfs54M3pibLw/640?wx_fmt=png&amp;from=appmsg" data-type="png" data-w="1080" src="https://mmbiz.qpic.cn/mmbiz_png/LjkGSkc5VwIgicG5W2sEU9UuWzL9U0x2Jt8Rsu3NTib2l0arlJQZ1ib7Ed1ag7czYnNChxPFtZOictFhfs54M3pibLw/640?wx_fmt=png&amp;from=appmsg"></p><p><br></p><h3 data-tool="mdnice编辑器"><span></span><span>Conclusion</span><span></span></h3><p data-tool="mdnice编辑器">STAligner aims to identify consensus spatial domains across slices, which is different from the goal of the non-integration methods BayesSpace[^11], SpaGCN10 and STAGATE12.STAligner aims to identify consensus spatial domains across slices, which is different from the goal of the non-integration methods BayesSpace[^11], SpaGCN10 and STAGATE12These tools were mainly designed for spatial domain identification in each ST slice and cannot be directly used for simultaneous domain identification across slices.Spatial domain identification for a single slice will only reveal them separately without the domain alignment.They have to rely on some downstream analysis to determine such correspondence.The flexible transformation strategy can even be used to register slices from different samples while handling the anatomical differences across samples, build a common reference of organs of different individuals and construct ST atlases in the future[^44]</p><h2 data-tool="mdnice编辑器"><span></span><span>Study subjects</span><span></span></h2><h3 data-tool="mdnice编辑器"><span></span><span>3 datasets</span><span></span></h3><ul data-tool="mdnice编辑器"><li><section>These large-scale datasets require the integration method to be memory and time efficient. ==We recorded the computational time and graphics processing unit (GPU) memory usage cost by STAlinger on three datasets in Supplementary Fig. 14a==. When integrating the largest mouse hippocampus dataset with seven slices and more than 170,000 spots in total, STAligner is fast and takes only 8.5 min</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Data analysis</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>#method/wilcoxon_test</section></li><li><section>#method/pca</section></li><li><section>#method/icp_algorithm</section></li><li><section>#method/interquartile_range</section></li><li><section>#method/louvain_algorithm</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Findings</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>We demonstrate improved performance of STAligner on diverse datasets from different individuals, technologies, disease conditions and developmental stages</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Confirmation of earlier findings</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>Thus, STAligner could stitch multiple tissue slices into a large one for deciphering its global structure. Improved integration on slices from ==different platforms Here we applied STAligner to integrate two mouse olfactory bulb ST slices profiled by the Slide-seqV25 and Stereo-seq==[^6],[^9] platforms (Fig. 3a) with clear tissue structures (Fig. 3b) but substantial batch effects (Supplementary Fig. 7)</section></li><li><section>Each domain showed clear tissue structure and can be annotated based on the anatomic location using eHistology Kaufman annotations[^25] (Fig. 4a). To reconstruct the developmental trajectory of each tissue structure, ==we built connections between the same spatial domains in successive stages and generated an acyclic directed graph to reflect the ancestor==–descendant relationships (Fig. 4b)</section></li><li><section>We also found a mouse-specific region dermomyotome, which may be attributed to the difference in developmental speeds between mouse and human organogenesis. ==Identifying disease-related spatial domains Here we applied STAligner to two ST slices from normal==[^5] and Alzheimer’s disease (AD)[^8] mouse hippocampus tissue profiled by Slide-seqV2, and identified 16 distinct spatial domains</section></li><li><section>FCGR3B is a classical marker gene for natural killer cells or neutrophils[^38], indicating that domain 11 could be a region of cancer growth limited by immune activities. ==These results shown that STAligner can delineate the difference in tumor substructures across tumor samples==</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Counterpoint to earlier claims</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>We anticipate that future advancements in these directions would facilitate more comprehensive exploration of biological phenomena. ==STAligner aims to identify consensus spatial domains across slices, which is different from the goal of the non-integration methods BayesSpace==[^11], SpaGCN10 and STAGATE12</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Future work</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>Future work is expected to develop a spatialdomain-guided flexible registration method for 3D reconstruction. The flexible transformation strategy can even be used to register slices from different samples while handling the anatomical differences across samples, build a common reference of organs of different individuals and construct ST atlases in the future[^44].</section></li></ul><h2 data-tool="mdnice编辑器"><span></span><span>Data and code</span><span></span></h2><ul data-tool="mdnice编辑器"><li><section>Source data for Figs. 2–6 is available with this paper. The datasets analyzed in this study are all from publicly available datasets (Supplementary Table 1). Specifically, the human DLPFC dataset can be accessed in the spatialLIBD package (http://spatial.libd.org/spatialLIBD). The mouse olfactory bulb tissue data generated by Stereoseq and Slide-seqV2 platforms can be accessed from https://github.com/JinmiaoChenLab/SEDR_analyses and https://singlecell.broadinstitute.org/single_cell/study/SCP815, respectively. The mouse sagittal posterior and anterior brain data can be accessed at https://support.10xgenomics.com/spatial-gene-expression/datasets/1.0.0/ V1_Mouse_Brain_Sagittal_Posterior and https://support.10xgenomics.com/spatial-gene-expression/datasets/1.0.0/V1_Mouse_Brain_Sagittal_Anterior, respectively. The mouse embryo data can be accessed at https://db.cngb.org/stomics/mosta/. The human embryo data can be accessed at https://heoa.shinyapps.io/code/. The human breast cancer data can be accessed at https://cf.10xgenomics.com/samples/spatial-exp/1.0.0/V1_Breast_Cancer_Block_A_Section_1/V1_Breast_Cancer_Block_A_Section_1_web_summary.html, https://cf.10xgenomics.com/samples/spatial-exp/1.2.0/V1_Human_Invasive_Ductal_Carcinoma/V1_Human_Invasive_Ductal_Carcinoma_web_summary.html and https://cf.10xgenomics.com/samples/spatial-exp/2.0.0/CytAssist_FFPE_Human_Breast_Cancer/CytAssist_FFPE_Human_Breast_Cancer_web_summary.html. The normal and Alzheimer’s disease mouse hippocampus data can be accessed at https://singlecell.broadinstitute.org/single_cell/study/SCP815 and https://singlecell.broadinstitute.org/single_cell/study/SCP1663, respectively. The Slide-seq 3D mouse</section></li><li><section>An open-source Python implementation of the STAligner package is available at https://github.com/zhanglabtools/STAligner and https://doi.org/10.5281/zenodo.8315415 ref.[^56].</section></li><li><section>Supplementary information The online version contains supplementary material available at https://doi.org/10.1038/s43588-023-00528-w.</section></li></ul></section><p><br></p><h2 data-tool="mdnice编辑器"><span>作者之言：很抱歉由于小编学业繁重，断更太久，目前摸索出一条排版较快捷的方法，但是文章里还是涉及很多的错误，以及没有转成中文的格式，以后会逐步克服。对于新加的文献推荐栏目，只要是涉及到方法的文献，均可提供复现或代码讲解（点赞👍过100即可获取）</span></h2><h2 data-tool="mdnice编辑器"></h2><p><mp-style-type data-value="3"></mp-style-type></p></div>
